responses/content.js

```diff
@@ -69,19 +88,39 @@ function theAnswerGivenWasNotCorrect(people, articleData, scoreData){
 	} else {
 		scorePhrase += ` The record to beat today is ${scoreData.scoreMax}.`;
 	}
-
 	scorePhrase += ' Would you like to start a new game?'
 
-	return {
-		displayText : displayPhrase,
-		speech : voicePhrase,
-		ssml : `<speak>${voicePhrase}</speak>`,
-		article: articleData.title,
-		link: `https://ft.com/${articleData.id}`,
-		image: illustration,
-		score: scorePhrase
+	let responseObj = {
+		displayText : '',
+		speech      : '',
+		ssml        : '',
+		article     : '',
+		link        : '',
+		image       : '',
+		score       : scorePhrase
 	};
 
+	const basePhrase = `That was not the correct answer. ${people.expected.replace('people:', '')} was mentioned with ${people.seed.replace('people:', '')}`;
+
+	if (articleData == null) {
+		const specificPhrase = `${basePhrase} in an FT article.`;
+		responseObj.displayText = specificPhrase;
+		responseObj.speech      = specificPhrase;
+		responseObj.ssml        = `<speak>${specificPhrase}</speak>`
+		responseObj.article     = 'an FT article';
```

Comment:
Same remark as AnswerCorrect function, the `.article` property needs to says something else.